evennia/utils/test_resources.py

```diff
@@ -66,3 +66,85 @@ def tearDown(self):
         self.account.delete()
         self.account2.delete()
         super(EvenniaTest, self).tearDown()
+
+
+import unittest
+import inspect
+
+class TestCreateScript(EvenniaTest):
+
+    def test_create_script(self):
+        class TestScriptA(DefaultScript):
+            def at_script_creation(self):
+                self.key = 'test_script'
+                self.interval = 10
+                self.persistent = False
+
+        script = create.create_script(TestScriptA, key='test_script')
+        assert script is not None
+        assert script.interval == 10
+        assert script.key == 'test_script'
+        script.stop()
+
+    @unittest.expectedFailure
```

Comment:
You noticed I left this test as **failure expected**? 

I think it should be fixed and start passing instead.